submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -45,3 +44,29 @@ CREATE TABLE accidentCases(
 SELECT * FROM accidentCases;
 
 
+-- Task 3 Transportation Data Query.
+
+# 1. Total number of passengers who survived
+	select count(*) from accidentCases where accidentStatus > 0;
+	-- Answers => 342
+
+# 2. total number of unsurvived passengers
+  SELECT COUNT(*) FROM accidentCases WHERE accidentStatus = 0;
+  -- Answers => 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+	SELECT registeredPassengers.fullName, registeredPassengers.sex from registeredPassengers
+	inner join tripDetails on passengerId =tripDetails.passengerId where age  < '27'and embarkationPoint 
```

Comment:
you should specify the table_name.column_name for column names that appear on the same tables you are joining

Reply:
done. thanks